### PR TITLE
Update CSIMigration state in volumes.md

### PR DESCRIPTION
**Pull request by ramineni**
This PR updates CSIMigration feature state of openstack cinder

### CODE REVIEW

File: content/en/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -248,14 +248,14 @@ spec:
...
-{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
 
 The CSI Migration feature for Cinder, when enabled, shims all plugin operations
 from the existing in-tree plugin to the `cinder.csi.openstack.org` Container
 Storage Interface (CSI) Driver. In order to use this feature, the [Openstack Cinder CSI
 Driver](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-cinder-csi-plugin.md)
 must be installed on the cluster and the `CSIMigration` and `CSIMigrationOpenStack`
-Alpha features must be enabled.
+Beta features must be enabled.
```
Comment (kbhawkey) [inline content/en/docs/concepts/storage/volumes.md:258]: Hi @ramineni .
Thanks for opening this PR.
Checking if the _beta_ feature needs to be enabled? Or could this line be removed?

Comment (ramineni) [reply content/en/docs/concepts/storage/volumes.md:258]: @kbhawkey yes, its beta feature but off by default, so needs to be enabled. https://github.com/kubernetes/kubernetes/pull/85637

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-20907--kubernetes-io-master-staging.netlify.app/docs/concepts/storage/volumes/

**Comment by kbhawkey** [comment]
@ramineni . Changes look good.
/lgtm

**Comment by kbhawkey** [comment]
Thanks!
/approve

**Pull request merged**